cpp/examples/abm.cpp

```diff
@@ -25,11 +25,11 @@
  * The infection states are chosen randomly. They are distributed according to the probabilites set in the example.
  * @return random infection state
  */
-mio::abm::InfectionState determine_infection_state(double exposed, double infected, double carrier, double recovered)
+mio::abm::InfectionState determine_infection_state(double exposed, double infected_symptoms, double infected_no_symptoms, double recovered)
 {
-    double susceptible          = 1 - exposed - infected - carrier - recovered;
-    std::vector<double> weights = {susceptible,  exposed,       carrier,      infected / 2,
-                                   infected / 2, recovered / 2, recovered / 2};
+    double susceptible          = 1 - exposed - infected_symptoms - infected_no_symptoms - recovered;
+    std::vector<double> weights = {susceptible,  exposed,       infected_no_symptoms,      infected_symptoms / 2,
+                                   infected_symptoms / 2, recovered / 2, recovered / 2};
```

Comment:
*Error* ? I think the initialization is now wrong as we do not have two recovered states anymore. Please also check if vector length should is "hard-coded" and should be adapted somewhere else.

Reply:
Thank you for spotting this. Indeed, the recovered weights should be changed to only one value. What do you mean by your second sentence? The vector is local and only used to call the DiscreteDistribution, which works with any vector length.

Reply:
Corrected.